### PR TITLE
various 'minor' fixes :)

### PR DESCRIPTION
**Pull request by boegel**

### CODE REVIEW

File: _posts/2015-01-01-intro.md [hunk 1]
```diff
@@ -22,23 +22,37 @@ style: left
...
+but this is the first time that representatives from the HPC and Big Data communities
+are joining forces to bring both communities together with this devroom.
+
+- High Performance Computing (HPC) and Big Data are two important approaches to scientific computing.
+  HPC typically deals with smaller, highly structured data sets and huge amounts of computation while
+  Big Data, not surprisingly, deals with gigantic, unstructured data sets and focuses on the I/O
+  bottlenecks. With the Big Data trend unlocking access to an unprecedented amount of data, Data
+  Science has emerged to tackle the problem of creating processes and approaches to extracting
+  knowledge or insights from these data sets. Machine learning and predictive analytics are now the
+  predominant type of computation performed on large clusters.
```
Comment (boegel) [inline _posts/2015-01-01-intro.md:45]: I'm not sure this latter statement is true, at least not in a general sense.

@rvs: please reword?

File: _posts/2015-01-03-submissions.md [hunk 2]
```diff
@@ -7,25 +7,25 @@ fa-icon: file-text-o
...
 - Links to previous talks by the speaker
 
+Our intention is to have a full day of talks of about 20 minutes each, with an additional 5 minutes for questions by attendees.
+
 We would also like to note: 
 
 - Talks will be streamed live and will be recorded. By submitting a session, speakers agree to being recorded and having their talk made available.
-- All accepted talks will be about free and open source software. We highly discourage "marketing" talks.
-
-Furthermore, authors must identify any conflict of interest with the Devroom organizers.
```
Comment (boegel) [inline _posts/2015-01-03-submissions.md:28]: I dropped this intentionally. As organisers, we should be honest enough to make it very clear that we have a conflict of interest with certain submissions, and to review submissions objectively. FOSDEM is not an academic conference, so let's not try and scare people from submitting a talk proposal with a statement like this.

File: _posts/2015-01-03-submissions.md [hunk 3]
```diff
@@ -37,10 +37,9 @@ Furthermore, authors must identify any conflict of interest with the Devroom org
...
-and Data Science Devroom' as the 'Track'.
+<strong>When submitting your talk in Pentabarf, make sure to select the 'HPC, Big Data,
+and Data Science Devroom' as the 'Track'.</strong>
 
-If you already have a Pentabarf account from a previous year, please reuse it.
+<em>If you already have a Pentabarf account from a previous FOSDEM edition, please reuse it</em>.
 Create an account if, and only if, you don't have one from a previous year. If
-you have any issues with Pentabarf, do not despair: contact Ewan dot Higgs or
-Kenneth dot Hoste at the ghent university domain.
+you have any issues with Pentabarf, do not despair: contact hpc-bigdata-devroom [at] lists.fosdem.org .
```
Comment (boegel) [inline _posts/2015-01-03-submissions.md:45]: let's use our ML as a point of contact, so anyone can respond to questions?

Comment (ehiggs) [inline _posts/2015-01-03-submissions.md:45]: +1

File: _posts/2015-01-02-topics.md [hunk 4]
```diff
@@ -9,36 +9,45 @@ icon-title: true
 
 ### Topics of interest include, but are not limited to:
 
-- Architecture and design of HPC and Big Data systems
-- Architecture and design of ETL and data acquisition pipelines
+- Architecture and design of High Performance Computing (HPC) and Big Data systems
+- Architecture and design of ETL (Extract, Transform and Load) and data acquisition pipelines
```
Comment (ehiggs) [inline _posts/2015-01-02-topics.md]: Isn't it more consistent with the above to say  `Extract Transform and Load (ETL)` ?

Comment (boegel) [inline _posts/2015-01-02-topics.md]: maybe, but it looks weird with the 'and' after then :)

I'm happy to change it though

File: _posts/2015-01-01-intro.md (hunk 1 above)
Comment (rvs) [inline _posts/2015-01-01-intro.md:45]: Good point! How about: "Machine learning and predictive analytics algorithms have now joined the family of more traditional HPC algorithms and are pushing the requirements on cluster and data scalability"

Comment (boegel) [inline _posts/2015-01-01-intro.md:45]: OK, included. Thanks.

File: _posts/2015-01-02-topics.md (hunk 4 above)
Comment (boegel) [inline _posts/2015-01-02-topics.md]: changed it to (ETL)

**Comment by boegel** [comment]
@rvs, @ehiggs, @vasia: this should be good to go in now, please review/merge (I won't merge my own PR)

**Comment by vasia** [comment]
+1
thanks @boegel! (I don't think I have rights to merge :)).

**Comment by boegel** [comment]
@vasia: you do now :)

**Comment by vasia** [comment]
thanks, merged!

**Pull request merged**